packages/editor/src/components/editor-interface/index.js

```diff
@@ -207,7 +207,7 @@ export default function EditorInterface( {
 				isLargeViewport &&
 				showBlockBreadcrumbs &&
 				isRichEditingEnabled &&
-				blockEditorMode !== 'zoom-out' &&
+				blockEditorMode !== 'compose' &&
```

Comment:
I wonder if this condition should be removed or tied to zoom out instead. It seemed strange that the footer would go away when switching to compose mode.